tests/opentelemetry/flask_app.py

```diff
@@ -9,8 +8,6 @@
 from tests.webclient import PingFilter
 
 
-# Enable ddtrace context management and set the datadog tracer provider
-os.environ["OTEL_PYTHON_CONTEXT"] = "ddcontextvars_context"
```

Comment:
We can get rid of this because we changed the other envar in the riotfile.py?

Reply:
answered my own question, the answer is yes: https://github.com/DataDog/dd-trace-py/blob/10a31be1023454e0ce7fcbfa069a86268b003d21/ddtrace/settings/config.py#L574